api/v1alpha1/packagebundle.go

```diff
@@ -152,6 +165,29 @@ func (s BundlePackageSource) PackageMatches(other BundlePackageSource) bool {
 	return true
 }
 
+func (bp *BundlePackage) GetJsonSchema() ([]byte, error) {
+	// The package configuration is gzipped and base64 encoded
+	// When processing the configuration, the reverse occurs: base64 decode, then unzip
+	configuration := bp.Source.Versions[0].Schema
```

Comment:
outside of the scope of this PR, but this will obviously not work if they are installing Versions[1]

Reply:
Currently I am not aware of any mechanism to provide the version of packages. Maybe once adding pinning to packages, we can decide to update this.

Reply:
PackageVersion